react/features/base/config/configType.ts

```diff
@@ -462,6 +462,7 @@ export interface IConfig {
         hideExtraJoinButtons?: Array<string>;
     };
     prejoinPageEnabled?: boolean;
+    reactionsButtonEnabled?: boolean;
```

Comment:
Can we use `toolbarButtons` instead? That is how all buttons are currently configured, why does this one need a dedicated config option?

Reply:
I was considering this since you initially proposed it and to be honest I really like the idea. The downside is that on mobile we don't use them and that's why I decided to go with the config option. Maybe we can start using the `config.toolbarButtons` on mobile and initially use it only for the reactions button. WDYT?

Reply:
We can try that! I thought this PR didn't tackle mobile yet though.

Reply:
It doesn't but there will be a follow up one for mobile.

Will do the change!